aleph-client/src/lib.rs

```diff
@@ -18,8 +18,12 @@ pub use session::{
 pub use staking::{
     bond as staking_bond, force_new_era as staking_force_new_era,
     set_staking_limit as staking_set_staking_limits, validate as staking_validate,
+    wait_for_full_era_completion as staking_wait_for_full_era_completion,
```

Comment:
I'm not so convinced to prefixing such methods; Ideally, I would keep these library modules public, but this is against proposed ADR...

Reply:
I would prefix them only if they are not readable without the prefix – here I would just `pub use` the original name.

Reply:
Okay, will do

Reply:
Done